main.cpp

```diff
@@ -832,11 +832,18 @@ class Framework {
           lv_obj_del(screen_off_label);
         }
       }
+
       if (print_memory_usage) {
-        // print free memory with the knowledge that 14KiB RAM is the actual PineTime-Memory
         lv_mem_monitor(&mem_mon);
-        printf("actual free_size = %d\n", int64_t(mem_mon.free_size) - (LV_MEM_SIZE - 14U*1024U));
+        if (mem_mon.free_size != mem_mon_last_free_size) {
+          uint32_t pinetime_lvgl_memory = 14U*1024U; // 14KiB is the LVGL memory size used in InfiniTime
```

Comment:
the compiler most probably does this, but please make this constant a `constexpr` to explicitly mark it as compile time constant